certgen/certProcessor/src/main/java/org/incredible/certProcessor/views/HTMLVarResolver.java

```diff
@@ -107,4 +109,21 @@ public String getExpiryDate() {
         return certificateExtension.getExpires();
     }
 
+    public Map<String, String> getCertMetaData() {
```

Comment:
Lets add - issuer name, recipient id as well to this map.

Reply:
done